templates/frontend/objects/chapter.tpl

```diff
@@ -0,0 +1,501 @@
+ * consists of the following:
+ *
+ * <!-- Wrapper class which provides proper spacing between components -->
+ * <div class="item">
+ *     <!-- Title/value combination -->
+ *     <div class="label">Abstract</div>
+ *     <div class="value">Value</div>
+ * </div>
+ *
+ * All styling should be applied by class name, so that titles may use heading
+ * elements (eg, <h3>) or any element required.
+ *
+ * <!-- Example: component with multiple title/value combinations -->
+ * <div class="item">
+ *     <div class="sub_item">
+ *         <div class="label">DOI</div>
+ *         <div class="value">12345678</div>
+ *     </div>
+ *     <div class="sub_item">
+ *         <div class="label">Published Date</div>
+ *         <div class="value">2015-01-01</div>
+ *     </div>
+ * </div>
+ *
+ * <!-- Example: component with no title -->
+ * <div class="item">
+ *     <div class="value">Whatever you'd like</div>
+ * </div>
+ *
+ * Core components are produced manually below, but can also be added via
+ * plugins using the hooks provided:
+ *
+ * Templates::Catalog::Chapter::Main
+ * Templates::Catalog::Chapter::Details
+ *
+ * @uses $currentPress Press The press currently being viewed
+ * @uses $monograph Monograph The monograph to be displayed
+ * @uses $publication Publication The publication (version) that is being displayed
+ * @uses $currentPublication Publication The latest published version of this monograph
+ * @uses $dateFormatShort string String defining date format that is passed to
+ *       smarty template function
+ * @uses $series Series The series this monograph is assigned to, if any.
+ * @uses $publicationFormats array List of PublicationFormat objects to display
+ * @uses $availableFiles array List of available MonographFiles
+ * @uses $ccLicenseBadge string An HTML string containing a CC license image and
+ *       text. Only appears when license URL matches a known CC license.
+ * @uses $chapter Chapter The chapter to be displayed.
+ * @uses $chapterAuthors array List of authors associated with this chapter.
+ * @uses $sourceChapter Chapter Earliest published version of the requested chapter.
+ * @uses $datePublished	date Date this chapter was published.
+ * @uses $firstDatePublished date Earliest chapter publishing date.
+ * @uses $chapterPublicationIds array List of publication ids containing this chapter.
+ *}
+<div class="obj_monograph_full obj_chapter">
+
+	{* Notification that this is an old version *}
+	{if $currentPublication->getID() !== $publication->getId() && !$hasVersionChanged}
+		<div class="cmp_notification notice">
+			{capture assign="latestVersionUrl"}{url page="catalog" op="book" path=$monograph->getBestId()|to_array:"chapter":$chapter->getSourceChapterId()}{/capture}
+
+			{translate key="submission.outdatedVersion"
+				datePublished=$publication->getData('datePublished')|date_format:$dateFormatShort
+				urlRecentVersion=$latestVersionUrl|escape
+			}
+		</div>
```

Comment:
When a chapter has been deleted in a more recent version this can lead to a 404 URL. The case I found was:

1. Create a chapter with a landing page.
2. Publish book.
3. Create a version of the book.
4. Delete the chapter.
5. Publish new version.
6. Go to old version of the book in the press catalog.
7. Go to old version of the chapter.
8. See the message that this is an outdated version.
9. Click the link and see a 404.

This happens because the link tries to go to the latest version, but the latest version is actually the current version. In such cases, where this is the latest version of the chapter but not of the book, I think we should use the same message but change the link to go to the book page.